client/client.go

```diff
@@ -97,6 +101,22 @@ func (c *client) Query() QueryClient {
 	return c.qclient
 }
 
+func (c *client) NodeSyncInfo(ctx context.Context) (*tmrpc.SyncInfo, error) {
+	node, err := c.cctx.GetNode()
+	if err != nil {
+		return nil, err
+	}
+
+	status, err := node.Status(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	info := status.SyncInfo
+
+	return &info, nil
```

Comment:
Return this by value? I can't see why it needs to be a pointer.

Reply:
main reason is mocks. need to simulate block height is going up
```
	client.On("NodeSyncInfo", mock.Anything).Run(func(args mock.Arguments) {
 		nodeSyncInfo.LatestBlockHeight = int64(time.Now().Sub(startedAt) / blockPeriod)
 	}).Return(nodeSyncInfo, nil)
```